### PR TITLE
BAL-2-GP-Login-Errors

### PR DESCRIPTION
**Pull request by gperissetcelteeka**
# Login errors

## :recycle: Current situation & Problem
*Describe the current situation (if possible, with an exemplary (or real) code snippet and/or where this is used)*
*Please link any open issue that is addressed with this PR*
#2 

## :bulb: Proposed solution
*Describe the solution and how this affects the project and internal structure*

## :gear: Release Notes 
*Add a summary of the feature and possible migration guides if this is a breaking change so this section can be added to the release notes.*
*Include code snippets that provide examples of the feature implemented if it appends or changes the public interface.*

## :heavy_plus_sign: Additional Information
*Provide some additional information if possible*

### Related PRs
*Reference the related PRs*

### Testing
*Are there tests included? If yes, which situations are tested, and which corner cases are missing?*

### Reviewer Nudging
*Where should the reviewer start? Where is a good entry point?*

### Code of Conduct & Contributing Guidelines 

By submitting creating this pull request, you agree to follow our [Code of Conduct](https://github.com/CS342/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/CS342/.github/blob/main/CONTRIBUTING.md):
- [x] I agree to follow the [Code of Conduct](https://github.com/CS342/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/CS342/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: Balance/Login/LoginView.swift [hunk 1]
```diff
@@ -19,16 +19,17 @@ struct LoginView: View {
...
-    
+    @State private var alertMessage: String = ""
+    @State private var showingAlert = false
+
     var body: some View {
         Group {
             if !authModel.isLoggedIn {
-                signInView
-                    .adaptsToKeyboard()
+                    signInView
```
Comment (sebastianricaldoni) [inline Balance/Login/LoginView.swift:31]: Gonza, not sure this will be a problem but just flagging you this variable name `signInView` since I saw you did some sort of renaming from `SignIn` to `SignUp`? 
@gperisset

File: Balance/Login/LoginView.swift [hunk 2]
```diff
@@ -47,7 +48,18 @@ struct LoginView: View {
...
+        }
         .background(backgroundColor)
+        .alert(alertMessage, isPresented: $showingAlert) {
+            Button("OK", role: .cancel) {
+                authModel.authError = ""
+            }
+        }
     }
     
     var signInView: some View {
```
Comment (sebastianricaldoni) [inline Balance/Login/LoginView.swift:65]: Ditto

**Review by sebastianricaldoni — APPROVED** [review]
LGTM. I just left a comment in case we missed renaming a variable.
